src/edu/csus/ecs/pc2/core/PacketHandler.java

```diff
@@ -1177,6 +1177,9 @@ private Profile cloneContest (Packet packet, ProfileCloneSettings settings, bool
         Profile newProfile = new Profile(settings.getName());
         newProfile.setDescription(settings.getDescription());
         newProfile.setSiteNumber(contest.getSiteNumber());
+        if (!StringUtilities.isEmpty(packet.getContestIdentifier())) {
```

Comment:
I guess it does make more sense to copy the contest ID here than in `InternalContest `since `InternalContest `is only a single implementation of `IInternalContest`, and we want the ID copied for `IInternalContest` implementation.  Please confirm this was your thinking in doing it this way.  The suggesting I put in the issue, namely, update it in `InternalContest`, appears to be insufficient.